src/applications/vaos/components/cancel/CancelAppointmentModal.jsx

```diff
@@ -26,7 +29,7 @@ export default function CancelAppointmentModal(props) {
     return null;
   }
 
-  if (appointmentToCancel.vaos?.videoType) {
+  if (isVideoAppointment(appointmentToCancel)) {
```

Comment:
Now that this returns true for both booked and pending appointments, I think we might need to also check that the status is `booked` here.

Reply:
Added check for `booked`